dotcom-rendering/src/web/components/DecideOnwards.tsx

```diff
@@ -0,0 +1,53 @@
+import { decideTrail } from '../lib/decideTrail';
+import { Carousel } from './Carousel.importable';
+import { ElementContainer } from './ElementContainer';
+import { FetchOnwardsData } from './FetchOnwardsData.importable';
+import { Island } from './Island';
+
+export const DecideOnwards = ({
+	onwards,
+	format,
+}: {
+	onwards: CAPIOnwardsType[];
+	format: ArticleFormat;
+}) => (
+	<>
+		{onwards.map(({ heading, trails, onwardsType, url }) => {
+			if (trails.length > 0) {
+				return (
+					<ElementContainer key={onwardsType}>
+						<Island deferUntil="visible">
+							<Carousel
+								heading={heading}
+								trails={trails.map(decideTrail)}
+								onwardsType={onwardsType}
+								format={format}
+							/>
+						</Island>
+					</ElementContainer>
+				);
+			}
+
+			if (url) {
+				return (
+					<ElementContainer key={onwardsType}>
+						<Island
+							clientOnly={true}
+							deferUntil="visible"
+							placeholderHeight={600}
```

Comment:
How did we decide on the placeholder height?

Reply:
This is the existing value (not added here) but it's essentially a best guess based on how cards render. Longer or shorter headlines will change the actual height